src/main/java/net/masterthought/cucumber/json/support/StepAdapter.java

```diff
@@ -0,0 +1,33 @@
+package net.masterthought.cucumber.json.support;
+
+import com.google.gson.*;
+import net.masterthought.cucumber.json.Step;
+
+import java.lang.reflect.Type;
+
+
+public class StepAdapter implements JsonDeserializer<Step> {
+
+    @Override
+    public Step deserialize(JsonElement jsonElement, Type type,
+                            JsonDeserializationContext jsonDeserializationContext)
+            throws JsonParseException {
+
+        JsonObject jo = (JsonObject) jsonElement;
+        JsonElement output = jo.get("output");
```

Comment:
export this as private final String OUTPUT_KEY = "output"

so you can use this here and below